frontend/src/views/files/Preview.vue

```diff
@@ -341,7 +341,7 @@ export default {
       this.$store.commit("updateRequest", {});
 
       let uri = url.removeLastDir(this.$route.path) + "/";
-      this.$router.push({ path: uri });
+      this.$router.replace({ path: uri });
```

Comment:
Assuming that the current directory is already on the history, `replace` will insert another history entry for the same directory, making users to press the back button twice to go back from that directory. Use the `push` method instead to avoid a duplicated history entry.

Reply:
<img width="439" alt="1" src="https://user-images.githubusercontent.com/76441520/134623919-adaa4948-1e6b-49b8-97c6-984315777ae3.png">
No. Look at this pic, if we are in the preview layer, the background layer(listingitem) 's link is not a folder, it's a file's link.    If we use push here, we click the "close" button to trigger this close(), , it will go to folder "b". But after that, we click "back" button of the browser, it will revisit the c.jpg, the pic will show up again.

Reply:
If we wanna make it perfect, we need to change some other files which also have "@router.push" except "listingitem.vue". 
<img width="664" alt="2021-09-24 133649" src="https://user-images.githubusercontent.com/76441520/134625203-827fcb99-b2b1-4791-8eba-ffa9660222a3.png">

When I got time. I will test it one by one. 
But only changing the file "preview.vue" have already avoided most of the "back" clicks.

Reply:
oh, I tested it. and different phone browser has different performance. For some browsers, it will revisit the c.jpg, the pic will show up again. but Chrome and Edge work in the way as you said. I will fix it.